docs/api.rst

```diff
@@ -29,11 +29,17 @@ You can read more about `Facebook's Graph API here`_.
   ``2.5``, ``2.6``, and ``2.7``. The default version is ``2.1`` and is used if
   the version keyword argument is not provided.
 * ``proxies`` - A ``dict`` with proxy-settings that Requests should use. `See Requests documentation`_.
+* ``app_secret`` (optional) The secret key of your app. If both ``app_secret``
+  and ``access_token`` are present this will be used to compute an
+  `appsecret_proof`_ *(This prevents bad guys from making API calls with your
+  access tokens from their servers.)*
+
```

Comment:
It would be great if this could follow the same format as the other function variables in the API reference (using `-`, mentioning the variable format, making the link to the Facebook documentation explicit, etc.). I also don't think the italicized explanation is necessary.